fili-core/src/main/java/com/yahoo/bard/webservice/data/config/table/MetricUnionCompositeTableDefinition.java

```diff
@@ -79,12 +86,33 @@ public MetricUnionCompositeTableDefinition(
 
     @Override
     public ConfigPhysicalTable build(ResourceDictionaries dictionaries, DataSourceMetadataService metadataService) {
```

Comment:
Can we add a test to test this build logic correctly builds the availabilitiesToMetricNames map?